src/icons/Wifi/Wifi.native.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Svg, { Path, G, Defs, ClipPath } from 'react-native-svg';
+import { getThemeColors } from '../../_helpers/theme';
+import colors from '../../tokens/colors';
```

Comment:
rename this as `WiFiOn`?

Reply:
Let's keep it WiFi only?, unless the icon explicitly expresses an action, like in WiFiOff?